ShareBook/ShareBook.Service/Email/EmailService.cs

```diff
@@ -56,5 +76,13 @@ private MimeMessage FormatEmail(string emailRecipient, string nameRecipient, str
             };
             return message;
         }
+
+        private List<User> GetAdminUsers()
```

Comment:
Raffa acho que ficou bom não tem como fugir não! Mas aqui já poderia pegar os e-mails diretamente, com um select ! Eai la em cima você não precisa do for saca ? Faz sentido ?

Reply:
Faz total sentido. Até poupa um pouco o banco. E de pouco em pouco a gente ganha escala. Show. Deixa comigo que vou ajustar agora mesmo.